### PR TITLE
docs: add showcase

### LINKED ISSUE #116: Write a guide about showcase
`showcase` is a server setting that remains undocumented. An attempt was made here: https://github.com/tailcallhq/tailcallhq.github.io/pull/101 but never reached completion.

- Update `@server` with the `showcase` setting.
- Why is it available ~ To try out random configurations hosted
- Performance considerations with using showcase (it's slow)

### PR DESCRIPTION
**Pull request by neo773**
This PR adds documentation for `showcase` option on `#server` directive

/claim #116
/closes #116

### CODE REVIEW

**Comment by tusharmath** [comment]
@neo773 This is great, but the issue requires us to write a guide about the showcase feature.

**Comment by neo773** [comment]
@tusharmath 
I must’ve misinterpreted it, the issue began with “update server.md”

**Comment by mogery** [comment]
Should probably also mention that it can be a security risk wrt file and env variable access.

**Comment by tusharmath** [comment]
@neo773 are you working on this?

**Comment by neo773** [comment]
@tusharmath 
Yes

File: docs/operators/server.md [hunk 1]
```diff
@@ -215,6 +215,37 @@ schema @server(key: "./key.pem") {
...
+#### Example Usage:
+
+```graphql showLineNumbers
+schema @server(showcase: true) {
+  query: Query
+}
+
+type User {
+  not_id: Int
+  not_name: String
```
Comment (tusharmath) [inline docs/operators/server.md]: Let's use camel case everywhere.

**Comment by neo773** [comment]
@tusharmath 
Pushed changes

**Review by tusharmath — APPROVED** [review]

**Pull request merged**